### PR TITLE
Release 0.5.2

### PR DESCRIPTION
**Pull request by mortendahl**
Release plus smaller changes required to make it go through

### CODE REVIEW

**Comment by mortendahl** [comment]
TFE tests run on rc2

File: docs/source/conf.py [hunk 1]
```diff
@@ -24,9 +24,9 @@
 author = 'The tf-encrypted Contributors'
 
 # The short X.Y version
-version = '0.5.1'
+version = '0.5.2'
 # The full version, including alpha/beta/rc tags
-release = '0.5.1'
+release = '0.5.2-rc2'
```
Comment (jvmancuso) [inline docs/source/conf.py]: if this is the commit for the 0.5.2 release, I believe the rc tags need to be removed in each of these files -- hopefully @ianlivingstone can confirm here

Comment (ianlivingstone) [reply docs/source/conf.py]: Yep! They need to be updated for each release. Ideally, these values should be centralized in one file or templated and derived at the time of the release so no one has to think about it.

Comment (mortendahl) [reply docs/source/conf.py]: kept them in in case we needed a rc3 -- if none of you think so i'll prepare the final release

**Pull request merged**